.github/workflows/bleeding-edge.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI (bleeding edge)
+# this workflow is heavily inspired from pandas, see
+# https://github.com/pandas-dev/pandas/blob/master/.github/workflows/python-dev.yml
+
+# goals: check stability against
+# - dev version of Python, numpy, and matplotlib
+# - building with future pip default options
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    # run this every 6 hours (time is UTC)
+    - cron: '0 3,9,15,21 * * *'
```

Comment:
Let's think about rainforests and our grandchildren. Once a day is more than plenty...

Reply:
You're right. I got excited, but I'll fix this